crates/transaction-pool/src/blobstore/mod.rs

```diff
@@ -34,11 +34,12 @@ pub trait BlobStore: fmt::Debug + Send + Sync + 'static {
     /// Deletes multiple blob sidecars from the store
     fn delete_all(&self, txs: Vec<B256>) -> Result<(), BlobStoreError>;
 
-    /// A maintenance function that can be called periodically to clean up the blob store.
+    /// A maintenance function that can be called periodically to clean up the blob store, returns
+    /// the number of successfully deleted blobs and the number of failed deletions.
     ///
     /// This is intended to be called in the background to clean up any old or unused data, in case
     /// the store uses deferred cleanup: [DiskFileBlobStore]
-    fn cleanup(&self);
+    fn cleanup(&self) -> (usize, usize);
```

Comment:
I'd like to use a struct for this

Reply:
Thanks for the advice, I'll adjust it asap

Reply:
@mattsse PTAL, thanks. BTW, the `InMemoryBlobStore` deletes the blob txs in sync mode, so should we adjust it into async mode or leave it as is?